### PR TITLE
Merge with stomp config

### PR DESCRIPTION
**Pull request by mjpatell**

### CODE REVIEW

File: cob_bringup/robots/raw3-1-calib.launch [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0"?>
+<launch>
```
Comment (fmessmer) [inline cob_bringup/robots/raw3-1-calib.launch]: what's the difference between ```raw3-1``` and ```raw3-1-calib```?
there should only be one setup for each physically existing robot

File: cob_bringup/robots/raw3-1-calib.xml [hunk 2]
```diff
@@ -0,0 +1,119 @@
+<?xml version="1.0"?>
```
Comment (fmessmer) [inline cob_bringup/robots/raw3-1-calib.xml]: what's the difference between ```raw3-1``` and ```raw3-1-calib```?
there should only be one setup for each physically existing robot

File: cob_hardware_config/robots/raw3-1/config/arm_modern_controller.yaml [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+max_command_silence: 0.5
+
+# Publish all joint states ----------------------------------
+joint_state_controller:
+   type:         joint_state_controller/JointStateController
+   publish_rate: 125
+
+# Publish wrench ----------------------------------
+force_torque_sensor_controller:
+   type:         force_torque_sensor_controller/ForceTorqueSensorController
```
Comment (fmessmer) [inline cob_hardware_config/robots/raw3-1/config/arm_modern_controller.yaml]: please add dependencies for the controller types to the ```package.xml``` as ```run_depend```

File: cob_hardware_config/robots/raw3-1/config/arm_modern_controller.yaml [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+   publish_rate: 125
+
+## joint trajectory controller
+joint_trajectory_controller:
+   type: position_controllers/JointTrajectoryController
```
Comment (fmessmer) [inline cob_hardware_config/robots/raw3-1/config/arm_modern_controller.yaml]: please add dependencies for the controller types to the ```package.xml``` as ```run_depend```

File: cob_moveit_config/robots/raw3-1/moveit/config/kinematics.yaml [hunk 5]
```diff
@@ -1,11 +1,11 @@
 arm:
-  kinematics_solver: kdl_kinematics_plugin/KDLKinematicsPlugin
+  kinematics_solver: trac_ik_kinematics_plugin/TRAC_IKKinematicsPlugin
```
Comment (fmessmer) [inline cob_moveit_config/robots/raw3-1/moveit/config/kinematics.yaml:2]: please add dependencies for the ik plugin type to the ```package.xml``` as ```run_depend```

File: cob_moveit_config/robots/raw3-1/moveit/config/kinematics.yaml [hunk 6]
```diff
@@ -1,11 +1,11 @@
...
   kinematics_solver_search_resolution: 0.005
-  kinematics_solver_timeout: 5
-  kinematics_solver_attempts: 3
+  kinematics_solver_timeout: 0.5
+  kinematics_solver_attempts: 5
   
 torso:
-  kinematics_solver: kdl_kinematics_plugin/KDLKinematicsPlugin
+  kinematics_solver: trac_ik_kinematics_plugin/TRAC_IKKinematicsPlugin
```
Comment (fmessmer) [inline cob_moveit_config/robots/raw3-1/moveit/config/kinematics.yaml:8]: please add dependencies for the ik plugin type to the ```package.xml``` as ```run_depend```

**Review by fmessmer — CHANGES_REQUESTED** [review]

File: cob_bringup/robots/raw3-1-calib.launch (hunk 1 above)
Comment (ipa-rmb) [reply cob_bringup/robots/raw3-1-calib.launch]: The calib can be removed, this was just a temporary setting of my student which we did not mean to add to github.

File: cob_bringup/robots/raw3-1-calib.xml (hunk 2 above)
Comment (ipa-rmb) [reply cob_bringup/robots/raw3-1-calib.xml]: see above

File: cob_moveit_config/package.xml [hunk 7]
```diff
@@ -13,5 +13,6 @@
   <buildtool_depend>catkin</buildtool_depend>
 
   <exec_depend>xacro</exec_depend>
+  <exec_depend>trac_ik</exec_depend>
```
Comment (fmessmer) [inline cob_moveit_config/package.xml]: sorry, I guess the convention is to keep dependencies of `cob_moveit_config` simple...but add them to `cob_moveit_bringup`...I did this in https://github.com/ipa320/cob_manipulation/pull/116...so please remove the dependency here...sorry

**Review by fmessmer — CHANGES_REQUESTED** [review]

File: cob_moveit_config/robots/raw3-1/moveit/config/kinematics.yaml (hunk 5 above)
Comment (fmessmer) [reply cob_moveit_config/robots/raw3-1/moveit/config/kinematics.yaml:2]: done in https://github.com/ipa320/cob_manipulation/pull/116

File: cob_moveit_config/robots/raw3-1/moveit/config/kinematics.yaml (hunk 6 above)
Comment (fmessmer) [reply cob_moveit_config/robots/raw3-1/moveit/config/kinematics.yaml:8]: done in https://github.com/ipa320/cob_manipulation/pull/116

File: cob_moveit_config/robots/raw3-1/moveit/config/constraints_limit.yaml [hunk 8]
```diff
@@ -0,0 +1,90 @@
+
```
Comment (fmessmer) [inline cob_moveit_config/robots/raw3-1/moveit/config/constraints_limit.yaml:1]: where is this file used?
please remove this file if it was also only used for testing purposes as in *_calib...

**Review by fmessmer — CHANGES_REQUESTED** [review]

File: cob_moveit_config/robots/raw3-1/moveit/config/constraints_limit.yaml (hunk 8 above)
Comment (mjpatell) [reply cob_moveit_config/robots/raw3-1/moveit/config/constraints_limit.yaml:1]: I used this file for joint constraints in ipa_order_picking_arm_planning package. I need this file.

**Comment by fmessmer** [comment]
we won't merge this into `cob_robots` see https://github.com/ipa320/cob_robots/pull/683#issue-243463031

@ipa-mjp 
please put the respective changes into your scenatio-specific moveit_config package within `ipa_order_picking`

**Pull request closed without merging**